app/views/shared/_footer_lite.html.erb

```diff
@@ -24,18 +24,18 @@
     </div>
   <% end %>
   <div class='container padding-y-1 padding-x-2 desktop:padding-x-0 <%= 'tablet:padding-y-0' if show_language_dropdown %>'>
-    <div class='flex flex-align-center flex-justify-center'>
-      <div class='flex flex-auto flex-first'>
```

Comment:
`flex-first` was removed without replacement here, because there is no equivalent, but more importantly, it doesn't appear to be necessary. [The style effectively applies `order: -1`](https://github.com/basscss/basscss-sass/blob/8edc9d2f0df5aec43dab31db3fcfbce7e35a0a7e/_flex-object.scss#L96). Aside from being a [discouraged style](https://developer.mozilla.org/en-US/docs/Web/CSS/order#accessibility_concerns), it's also not really necessary here, because it's already the first child in its container.